PD-115 Create Update HA SCALE Article (#1747)
truenas/documentation: content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md

To see user accounts, go to **Credentials > Local Users**.

![AllUsersScreenSCALE](/images/SCALE/22.12/AllUsersScreenSCALE.png "Local User non-Built-in Accounts") 

TrueNAS hides all built-in users (except root) by default. Click the toggle **Show Built-In Users** to see all built-in users.

## Creating an Admin User Account

> SCALE has implemented rootless log in. All systems should create and begin using an admin user as a replacement for the root user.  

Reviewer: ```suggestion
SCALE has implemented rootless login. All systems should create and use an admin user to replace the root user.  
```

Author: Claire, I'm not sure how your article got into my PR. Would you review this open comment and either accept or change the suggestion made by Tony.
